pkg/proto/agent.proto

```diff
@@ -20,9 +22,20 @@ message InitRequest {
 
 message CreatEnvRequest {
     string eventTag = 1;
-    repeated VmConfig vms = 2;
-    int32 labAmount = 3;
-    repeated string exercises = 4;
+    int32 envType = 2;
```

Comment:
I know it is nicer to have them in order like this, but proto best practice is never to change numbers on fields as this breaks the backwards compatibility

Reply:
Since nothing is currently dependent on the agent yet, i guess it is not currently a problem. However will keep this in mind as i did not think of it  :)!